### DISCUSSION TITLE
Remove Operator Command Shortcut Favorites

### DISCUSSION THREAD

**Discussion by KuwuroUsagi** [shedaniel/RoughlyEnoughItems#1043] [Q & A] [answered]
Simple Q hopefully a simple A.

Would like to remove some of these buttons pictured below so I don't accidentally click them while I'm playing and swinging my mouse around.
I see how to add more buttons, but not how to remove these seemingly "default" ones in the pack I'm using. 

![image](https://user-images.githubusercontent.com/6632311/182034347-ab536565-e335-48d8-8a30-b1b02f3b9b7e.png)

**Comment by shedaniel** [comment, marked as answer]
Press A on them? FTB Library buttons should be removable in 1.19 but not in 1.18, so check that

**Reply by KuwuroUsagi** [reply]
> Not in 1.18

Thank you!

**Reply by MaxNeedsSnacks** [reply]
(They're now removable in 1.18 as well)
